src/main/java/frc/robot/Robot.java

```diff
@@ -63,6 +95,40 @@ public void robotPeriodic()
     //SmartDashboard.putBoolean("Climbing Enabled ", currentlyClimbing);
     //SmartDashboard.putNumber("NavX Pitch:", navx.getPitch());
     // smart dashboard.put stats (navx connected bool, yaw, pitch, roll, climbing bool)
+
+    //Detects the colour, outputs it into the dashboard, and sets the ball colour ("ballIsBlue", "ballIsRed")
+    Color detectedColor = colorSensor1.getColor();
+
+    String colorString;
+    ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
+
+    if (match.color == kBlueTarget) {
+      colorString = "Blue";
+      ballIsBlue = true;
+      ballIsRed = false;
+    } else if (match.color == kRedTarget) {
+      colorString = "Red";
+      ballIsRed = true;
+      ballIsBlue = false;
```

Comment:
```suggestion
      ballIsBlue = false;
      ballIsRed = true;
```

Nit: switch the order of these two variables so it's the same as all the other `else if` statements. Makes it slightly easier to read the code.

FYI: "Nit" stands for "nitpick" and it means that the change I am suggesting isn't necessary (it's more of a preference type of thing). It just gives you some ideas on how things would have been done by other collaborators. Don't feel like you need to agree with my suggestion though! 😄